libcxxabi/test/test_demangle.pass.cpp

```diff
@@ -30290,124 +30305,106 @@ const char* invalid_cases[] =
 
 const unsigned NI = sizeof(invalid_cases) / sizeof(invalid_cases[0]);
 
-void test()
-{
-    std::size_t len = 0;
-    char* buf = nullptr;
-    bool failed = false;
-    for (unsigned i = 0; i < N; ++i)
-    {
-        int status;
-        char* demang =
-            __cxxabiv1::__cxa_demangle(cases[i][0], buf, &len, &status);
-        if (!demang || std::strcmp(demang, cases[i][1]) != 0)
-        {
-          std::fprintf(stderr, "ERROR demangling %s\n"
-                       "expected: %s\n"
-                       "got: %d,   %s\n",
-                       cases[i][0], cases[i][1], status,
-                       demang ? demang : "(null)");
-          failed = true;
-        }
-        if (demang)
-          buf = demang;
+void test() {
```

Comment:
Minor nit: can we clang-format this file separately? And also put a `// clang-format off` around the `invalid_cases` (like we do for the `cases` array). It's easier to copy-paste out the test-cases when they're not split across multiple lines

Reply:
Sorry, that was an accidental change, I have removed it. I don't intend to reformat the file.